boards/arm/mec2016evb_assy6797/doc/mec2016evb_assy6797.rst

```diff
@@ -0,0 +1,212 @@
+- I2C voltage translator
+- 10 SMBUS headers
+- 3 UDP I2C headers
+- VCI interface
+- 5 independent Hardware Driven PS/2 Ports
+- eSPI header
+- LPC sideband headers
+- 4 Breathing/Blinking LEDs
+- 2 Sockets for SPI NOR chips
+- One reset and VCC_PWRDGD pushbuttons
+
+For more information about the SOC please visit:
+
+- `MEC170x Reference Manual`_
+
+Supported Features
+==================
+
+The mec2016evb_assy6797 board configuration supports the following hardware
+features:
+
++-----------+------------+-------------------------------------+
+| Interface | Controller | Driver/Component                    |
++===========+============+=====================================+
+| NVIC      | on-chip    | nested vector interrupt controller  |
++-----------+------------+-------------------------------------+
+| SYSTICK   | on-chip    | systick                             |
++-----------+------------+-------------------------------------+
+| UART      | on-chip    | serial port                         |
++-----------+------------+-------------------------------------+
+
+Other hardware features are not currently supported by Zephyr (at the moment)
+
+The default configuration can be found in the Kconfig file:
+
+        ``boards/arm/mec2016evb_assy6797/mec2016evb_assy6797_defconfig``
+
+
+Connections and IOs
+===================
+
+Microchip to provide the schematic for this board.
+
+System Clock
+============
+
+The MEC1701 MCU is configured to use the 48Mhz internal oscilator with the
+on-chip PLL to generate a resulting EC clock rate of 12 MHz. See Processor clock
+control register (chapter 4 in user manual)
+
+Serial Port
+===========
+
+UART0 is configured for serial logs.
+
+Jumper settings
+***************
+
+Please follow the jumper settings below for in order to properly demo this
+board. Advanced users may deviate from this recomendation.
+
+Jump setting for MEC2016 EVB Assy 6797 Rev A1p0
+===============================================
+
+Power-related jumpers.
+
++-------+------+------+------+------+------+-------+-------+
+| JP31  | JP32 | JP35 | JP36 | JP49 | JP50 | JP54  | JP55  |
++=======+======+======+======+======+======+=======+=======+
+| 1-2   | 1-2  | 1-2  | 1-2  | 1-2  | 1-2  | 1-2   | 1-2   |
++-------+------+------+------+------+------+-------+-------+
+
++-------+-------+------+------+------+-------+
+| JP56  | JP57  | JP58 | JP60 | JP61 | JP102 |
++=======+=======+======+======+======+=======+
+| 1-2   | 1-2   | 1-2  | 1-2  | 1-2  | 2-3   |
++-------+-------+------+------+------+-------+
+
+This jumpers configure VCC Powergood, nRESETI and JTAG_STRAP respectively.
+
++-------+-------+------+
+| JP1   | JP2   | JP51 |
++=======+=======+======+
+| 1-2   | 1-2   | 2-3  |
++-------+-------+------+
+
+Each column of the following table illustrates how to enable UART0, JTAG,
+PVT SPI, SHD SPI and LED0-3 respectively.
+
++-------+-------+---------+--------+--------+
+| JP27  | JP10  | JP34    | JP75   | JP68   |
```

Comment:
Same comment here (add the names you mention above to this table heading)

Reply:
done